### ISSUE TITLE
acm.eff.pushEnabled neither true nor false until user toggles it

### ISSUE THREAD

**Issue by jeremytribby** [EFForg/actioncenter-mobile#53]
This started happening once I moved the toggle out of the header and into the settings tab, IIRC.

**Comment by mfb** [comment]
I don't think we have a problem here. It won't be set until someone sets it and that's ok.

If it's unset, we should assume it's on.
